### PR TITLE
Force enable-first-step=false

### PR DESCRIPTION
**Pull request by flomnes**

### CODE REVIEW

File: src/libs/antares/study/parameters/adq-patch-params.cpp [hunk 1]
```diff
@@ -128,8 +136,6 @@ void CurtailmentSharing::addProperties(IniFile::Section* section) const
 // ------------------------
 void AdqPatchParams::reset()
 {
-    enabled = false;
```
Comment (payetvin) [inline src/libs/antares/study/parameters/adq-patch-params.cpp]: This `enabled` var is related to adq patch, not the same as the one in local matching
This change is also present on develop, I'll do a PR to fix it

**Review by payetvin — DISMISSED** [review]

File: src/libs/antares/study/parameters/adq-patch-params.cpp (hunk 1 above)
Comment (payetvin) [reply src/libs/antares/study/parameters/adq-patch-params.cpp]: https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/2420

Comment (flomnes) [reply src/libs/antares/study/parameters/adq-patch-params.cpp]: done

File: src/libs/antares/study/parameters/adq-patch-params.cpp [hunk 2]
```diff
@@ -25,7 +25,15 @@ bool LocalMatching::updateFromKeyValue(const Yuni::String& key, const Yuni::Stri
...
     if (key == "enable-first-step")
-        return value.to<bool>(enabled);
+    {
+        if (value == "true")
+        {
+            logs.warning() << "Property enable-first-step has been disabled, it is known to cause "
+                              "errors and inconsistent results";
+        }
+        return true;
+    }
```
Comment (guilpier-code) [inline src/libs/antares/study/parameters/adq-patch-params.cpp]: _Cosmetic_

Or : 
```C++
if (key == "enable-first-step" && value == "true")
{
    logs.warning() << "Property enable-first-step has been disabled, it is known to cause "
                      "errors and inconsistent results"
    return true;
 }
```

File: src/libs/antares/study/parameters/adq-patch-params.h [hunk 3]
```diff
@@ -59,7 +59,7 @@ enum class AdqPatchPTO
 
 struct LocalMatching
```
Comment (guilpier-code) [inline src/libs/antares/study/parameters/adq-patch-params.h:60]: `struct LocalMatching` : do we keep this **struct** to have a minimal change ?

Comment (flomnes) [reply src/libs/antares/study/parameters/adq-patch-params.h:60]: yes

**Review by guilpier-code — APPROVED** [review]

**Pull request merged**